packages/publisher-dapp/src/Pages/Overview/index.js

```diff
@@ -1,11 +1,173 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import { withStyles } from "@material-ui/styles";
+import TextField from '@material-ui/core/TextField';
+import CheckIcon from "@material-ui/icons/CheckCircle";
 
-const Overview = () => {
-    return (
-        <div>
-            Overview
-        </div>
-    );
+import { useStyles } from "./styles";
+import StyledButton from "shared/dist/components/StyledButton";
+
+const Overview = ({ classes }) => {
+  return (
+  	<Grid container className={classes.overiewMainContainer}>
+  		<Grid item xs={12} sm={12} md={12} lg={12} className={classes.codeToCustomer}>
+        <Grid item xs={5} className={classes.codeToCustomerContent}>
+        	<Typography variant="h2">From Code to Customer</Typography>
+        	<Typography variant="body1">Join the Singularity Dev Publisher Program to reach customers around the world on the AI Marketplace for any platform,. You’ll also get access to beta software, advanced app capabilities, extensive beta testing tools, and app analytics.</Typography>
+        	<List>
```

Comment:
create ` content.js `  for list in the format
```
export const Banners = () => [
{title:string,
description:string,
list: Array<string>
img:string
},....]
```
map the array here. display alternative content `rtl`